lib/KettleRequest.js

```diff
@@ -233,8 +233,9 @@ kettle.request.http.handleFullRequest = function (request, fullRequestPromise, n
 
 /**
  * Send an error payload to the client if the request ends in error
- * @param  {Object} res an Express response object.
- * @param  {Object} error an error payload. Should include fields <code>statusCode</code> holding a numeric HTTP status code, and <code>message</code> holding an error message.
+ * @param {Object} res - an Express response object.
+ * @param {Object} error - an error payload. Should include fields <code>statusCode</code> holding a numeric HTTP status code, and <code>message</code> holding an error message.
```

Comment:
Another example of mixing cases in a single JSDocs block, this time the return description is capitalised and the param descriptions aren't.